project_euler/Problem_27/sol1.py

```diff
@@ -11,38 +11,52 @@
 
 import math
 
+
 def isPrime(k):
```

Comment:
Let’s use a Pythonic function name and add type hints: `def is_prime(k: int) -> bool`